src/schema/v2/me/myCollection.ts

```diff
@@ -86,8 +88,10 @@ export const MyCollection: GraphQLFieldConfig<any, ResolverContext> = {
       return null
     }
 
+    // We're setting the size to 100 here because we're going to filter and sort these artworks later manualy.
+    // The idea is to fetch (almost) all artworks and then filter/sort them in memory.
     const paginationArgs = options.sortByLastAuctionResultDate
-      ? { size: 100 }
+      ? { size: MAX_COLLECTION_SIZE }
```

Comment:
do we necessarily need to remove the other args here?
```suggestion
      ? { ...convertConnectionArgsToGravityArgs(options), size: MAX_COLLECTION_SIZE,  }
```

Reply:
This could be a potential solution for this bug: https://github.com/artsy/metaphysics/pull/4801. Let's keep the logic like it was in this chore and work on a solution in https://github.com/artsy/metaphysics/pull/4800.

Reply:
But yeah, sounds like a good idea to keep the other arguments. But I'm not sure if it has any negative consequences 🤔

Reply:
oki